lib/_http_outgoing.js

```diff
@@ -307,9 +307,9 @@ function storeHeader(self, state, field, value) {
 
 OutgoingMessage.prototype.setHeader = function(name, value) {
   if (typeof name !== 'string')
-    throw new TypeError('`name` should be a string in setHeader(name, value).');
+    throw new TypeError('"name" should be a string in setHeader(name, value).');
   if (value === undefined)
-    throw new Error('`value` required in setHeader("' + name + '", value).');
+    throw new Error('"value" required in setHeader("' + name + '", value).');
```

Comment:
`required` -> `is required`

I'd also say that we shouldn't replace "name" with the actual value of the variable